rnaseq-cgl-pipeline/Makefile

```diff
@@ -1,22 +1,28 @@
 # Definitions
-runtime_fullpath = $(realpath runtime)
-build_tool = runtime-container.DONE
-git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../rnaseq-cgl-pipeline | cut -f1 -d " ")
-name = quay.io/ucsc_cgl/rnaseq-cgl-pipeline
-tag = 732349f6dd0df3fb90fa9d7e6ee1e04a32e2e773--${git_commit}
+docker_versions = 1.7.0 1.7.1 1.8.0 1.8.1 1.9.0 1.9.1 1.10.0 1.10.1 1.10.3
+gen_directories = $(docker_versions:=.docker)
 
-build:
-	docker build -t ${name}:${tag} .
-	docker tag -f ${name}:${tag} ${name}:latest
-	touch ${build_tool}
+build: ${build_tool}
```

Comment:
@jvivian just wanted to confirm, we resolved this issue (`nothing to be done for build`) in our meeting on Friday, right? If so, let's delete this comment, as it has been addressed.